gradle.properties

```diff
@@ -11,4 +11,4 @@ org.gradle.jvmargs=-Xmx8g
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-android.enableD8MainDexList=false
+android.enableD8MainDexList=true
```

Comment:
Maybe we can remove this line. [Default option](https://android.googlesource.com/platform/tools/base/+/studio-master-dev/build-system/gradle-core/src/main/java/com/android/build/gradle/options/BooleanOption.java?autodive=0%2F%2F#48) seems to be set to `true`